packages/editor-ui/src/components/ParameterOptions.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+	<div class="parameter-options" v-if="displayOptionsComputed">
+		<el-dropdown trigger="click" @command="optionSelected" size="mini">
```

Comment:
```suggestion
		<el-dropdown trigger="click" @command="(opt) => $emit(opt)" size="mini">
```

we should not be passing a method down as prop.. we should be emitting an event